src/ExpressionEvaluator/Core/Source/ExpressionCompiler/MetadataUtilities.cs

```diff
@@ -178,9 +175,23 @@ internal static string GetFileNameWithoutExtension(string fileName)
             return fileName;
         }
 
+        private static byte[] GetWindowsProxyBytes()
+        {
+            var assembly = typeof(ExpressionCompiler).GetTypeInfo().Assembly;
+            using (var stream = assembly.GetManifestResourceStream("Microsoft.CodeAnalysis.ExpressionEvaluator.Resources.WindowsProxy.winmd"))
+            {
+                var bytes = new byte[stream.Length];
+                using (var memoryStream = new MemoryStream(bytes))
+                {
+                    stream.CopyTo(memoryStream);
+                }
+                return bytes;
```

Comment:
Does `MemoryStream` guarantee to preserve the content of the underlying array on `Dispose()`?